src/compiler/types.ts

```diff
@@ -3858,7 +3859,7 @@ namespace ts {
     export interface ExpandResult {
         fileNames: string[];
         wildcardDirectories: MapLike<WatchDirectoryFlags>;
-        spec: ConfigFileSpecs;
+        /* @internal */ spec: ConfigFileSpecs;

```

Comment:
Should this be optional? This member <a href="https://github.com/Microsoft/TypeScript/blob/ae3f29ade6a8af80a46ce55cf23dd44ebdfe4036/src/compiler/commandLineParser.ts#L1429">later feeds</a> into an optional `configFileSpec` declared as optional/internal here above at line 3835.

Reply:
In fact <a href="https://github.com/Microsoft/TypeScript/search?utf8=%E2%9C%93&q=ExpandResult">it looks like</a> whole `ExpandResult` is never exposed anywhere through any API, except for the declaration of the interface itself.

So the `/* @internal */` mark should go on whole `ExpandResult`.